src/main/resources/avro/variantmethods.avdl

```diff
@@ -51,6 +51,89 @@ SearchVariantSetsResponse searchVariantSets(
   /** This request maps to the body of `POST /variantsets/search` as JSON. */
   SearchVariantSetsRequest request) throws GAException;
 
+
+/******************  /variantsets/{id}/sequences/search  *********************/
+/**
+This request maps to the body of `POST /variantsets/{id}/sequences/search` as
+JSON.
+*/
+record SearchVariantSetSequencesRequest {
```

Comment:
It should definitely be sequence here, and later where I've accidentally talked about a `SearchVariantSetSegmentsRequest`.

The sequences are the actual components of the sequence graph, which is what you are asking about here. However, since sequences aren't represented by an Avro record type, we describe the sequences in the graph with a set of `Segment`s that cover them, one per sequence.

Segments do not have IDs; they are merely regions defined on sequences, possibly including the sequences' end joins.